charts/finrem-ccd-definitions/requirements.yaml

```diff
@@ -1,7 +1,7 @@
 dependencies:
   - name: nodejs
-    version: ~1.8.0
+    version: ~1.10.0
     repository: '@hmctspublic'
   - name: ccd
-    version: ~4.2.0
+    version: ~3.3.5
```

Comment:
why?

Reply:
same as div-ccd-definitions repo, updated version was causing issues previously, might not be anymore but don't think it's a bad idea to use the same version in both repos

Reply:
@jvictorcf I think you have updated it recently for a reason
<img width="495" alt="Screenshot 2020-04-27 at 16 57 00" src="https://user-images.githubusercontent.com/5647337/80386790-2d0e0c00-88a8-11ea-976e-ab977e21dbb0.png">

Reply:
That's fine. That was just some experimentation, @yarpo.
@shaunganley, could you please squash these commits? That would get rid of all failed attempts.